data/domains.csv

```diff
@@ -26,6 +26,7 @@ ABILITYONE.GOV,Federal Agency,Comm for People Who Are Blind/Severly Disabled,Arl
 JWOD.GOV,Federal Agency,Comm for People Who Are Blind/Severly Disabled,Arlington,VA
 CFTC.GOV,Federal Agency,Commodity Futures Trading Commission,Washington,DC
 SMARTCHECK.GOV,Federal Agency,Commodity Futures Trading Commission,Washington,DC
+WHISTLEBLOWER.GOV,Federal Agency,Commodity Futures Trading Commission,Washington,DC
```

Comment:
This `domains.csv` update to keep up with the `2016-01-19` data update, but is temporary -- we should be able to move to pulling this data right from GitHub, and/or Data.gov, before long.